src/amo/components/Addon/index.js

```diff
@@ -255,23 +255,41 @@ export class AddonBase extends React.Component {
 
   renderShowMoreCard() {
     const { addon, i18n } = this.props;
-    const addonType = addon ? addon.type : ADDON_TYPE_EXTENSION;
 
+    let title;
     const descriptionProps = {};
+
     if (addon) {
-      const description =
-        addon.description ? addon.description : addon.summary;
+      switch (addon.type) {
+        case ADDON_TYPE_DICT:
+          title = i18n.gettext('About this dictionary');
+          break;
+        case ADDON_TYPE_EXTENSION:
+          title = i18n.gettext('About this extension');
+          break;
+        case ADDON_TYPE_LANG:
+          title = i18n.gettext('About this language pack');
+          break;
+        case ADDON_TYPE_THEME:
```

Comment:
We should have another one for `OPENSEARCH`

Reply:
What the name of an `opensearch`?

Reply:
See: https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/RatingManager/index.js#L137.